networkmonitor/src/main/java/ca/rmen/android/networkmonitor/provider/NetMonDatabase.java

```diff
@@ -191,6 +192,9 @@
     private static final String SQL_UPDATE_TABLE_NETWORKMONITOR_V15_SERVICE_STATE = "ALTER TABLE " + NetMonColumns.TABLE_NAME + " ADD COLUMN "
             + NetMonColumns.SERVICE_STATE + " TEXT";
 
+    private static final String SQL_UPDATE_TABLE_NETWORKMONITOR_V15_DEVICE_SPEED = "ALTER TABLE " + NetMonColumns.TABLE_NAME + " ADD COLUMN "
```

Comment:
This new constant should have V16 instead of V15 in the field name, and it should be executed at the end of ```onUpgrade```, inside a block ```if (oldVersion < 16) {```